AudioManagerUtilities/include/CAmCommonAPIWrapper.h

```diff
@@ -133,24 +133,25 @@ class CAmCommonAPIWrapper
 	* @return Pointer to the socket handler.
 	*/
 	CAmSocketHandler *getSocketHandler() const { return mpSocketHandler; }
-#if COMMONAPI_VERSION_NUMBER >= 300
-	/**
-	* \brief Register stub objects.
-	*
-	* Example: std::shared_ptr<ConcreteStubClass> aStub;
-	* 		   registerService(	aStub, "local", "com.your_company.instance_name", "service-name");
-	*
-	* @param shStub: Shared pointer to a stub instance
-	* @param domain: A string with the domain name, usually "local"
-	* @param instance: Common-api instance string as example "com.your_company.instance_name"
-	* @param connectionId: A string connection id, which is used by CommonAPI to group applications
-	*
-	*/
-	template <class TStubImp> bool registerService(const std::shared_ptr<TStubImp> & shStub, const std::string & domain, const std::string & instance, const CommonAPI::ConnectionId_t & connectionId)
-	{
-		return mRuntime->registerService(domain, instance, shStub, connectionId);
-	}
-#endif
+
+    /** 
+    * \brief Deprecated method. This class is used only in single connection applications and no connectionId is needed. Instead you should use bool registerService(const std::shared_ptr<TStubImp> & shStub, const std::string & domain, const std::string & instance). 
+    * 
+    * 
+    * Example: std::shared_ptr<ConcreteStubClass> aStub; 
+    *        registerService(  aStub, "local", "com.your_company.instance_name", "service-name"); 
+    * 
+    * @param shStub: Shared pointer to a stub instance 
+    * @param domain: A string with the domain name, usually "local" 
+    * @param instance: Common-api instance string as example "com.your_company.instance_name" 
+    * @param connectionId: A string connection id, which is used by CommonAPI to group applications 
+    * 
+    */ 
+    template <class TStubImp> bool __attribute__((deprecated)) registerService(const std::shared_ptr<TStubImp> & shStub, const std::string & domain, const std::string & instance, const CommonAPI::ConnectionId_t __attribute__((__unused__)) & /*connectionId*/) 
```

Comment:
When reviewing it's very easy to miss the addition of 
```__attribute__((deprecated))```
(and a lot of work to read carefully if anything else has changed) because of all the indentation change.    Ideally only removing an #ifdef should not change any indentation.